emuu-client/src/components/common/UserProfileCard/UserProfileCard.js

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import "./UserProfileCard.scss";
+import { Avatar } from "@mui/material";
+
+import SubscribeButton from "../Button/Button";
+import { Link } from "react-router-dom";
+
+const UserProfileCard = ({
+  id,
+  profileImg,
+  username,
+  subscribersCount,
+  onClick,
+}) => {
+  const isAuthenticated = localStorage.getItem("auth");
+
+  console.log(isAuthenticated, "auth status");
+  return (
+    <div className="video__details__container">
+      <Link to="/UserProfile" className="avatar__container">
+        <Avatar
+          src={
+            profileImg
+              ? profileImg
+              : "https://wallpaperaccess.com/full/170249.jpg"
```

Comment:
Seems like a placeholder image, is the same stock image as the one when a new user registers?